extensions/blocks/podcast-player/view.js

```diff
@@ -0,0 +1,66 @@
+/* global _wpmejsSettings, MediaElementPlayer, jetpackPodcastPlayers */
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+
+const playerInstances = {};
```

Comment:
This may be a silly question, but why is this an `object` and not an `array`?

Reply:
Is that a common way of doing things in Gutenberg land? I'm used to doing things like:

```
const playerInstances = [];

// ready player one ;)
const playerOne = { id: 'player-one', tracks: [] };
playerInstances.push( playerOne );

// use a helper to make accessing the player instance easily
function getPlayerById( id ) {
  return playerInstances.find( p  => p.id === id );
}
```

Even though it's a bit more work to get the player we want, is there an added benefit to using it as an array? I totally see how using `playerInstances` as an object is a lot easier in our current context.

Reply:
It's all about the speed of accessing the instance needed. It's instant to access a property of an object. With an array, you need to loop through potentially all instances before you get the one you need. And not once, but every single time you need to access any instance.

This technique is heavily being used in the Redux world (thus Gutenberg too) where it's extremely common to use objects like this. All blocks in a document are stored this way for example because you need to access them often by id. This talks about it a bit and shows the complexity difference between an array and an object: https://medium.com/javascript-in-plain-english/https-medium-com-javascript-in-plain-english-why-you-should-use-an-object-not-an-array-for-lists-bee4a1fbc8bd

Reply:
On this context definitely `object` > `array`, imho.